vscode-dotnet-runtime-library/src/Utils/FileUtilities.ts

```diff
@@ -81,7 +81,7 @@ export class FileUtilities extends IFileUtilities
             try
             {
                 eventStream?.post(new FileToWipe(`The file ${f} may be deleted.`))
```

Comment:
It's logging the may-be-deleted message before it evaluates the file extension, in which case it may not be delete. The wording just seems a bit ambiguous since it sounds like it already determined that the deletion can proceed.

Would it make more sense to have an `else` statement that executes after the extension check and instead log when a file is not being deleted